Book-A-desk.Domain.Tests/Reservation.Validator.Tests.fs

```diff
@@ -123,4 +123,43 @@ let ``GIVEN A valid Book-A-Desk Reservation command, WHEN validating the command
     let result = BookADeskReservationValidator.validateCommand offices command aReservationAggregate
     match result with
     | Error _ -> failwith "Validation should have succeeded"
-    | Ok _ -> ()
\ No newline at end of file
+    | Ok _ -> ()
+    
+[<Fact>]
+let ``GIVEN A Book-A-Desk Reservation command with the user not already booked, WHEN validating the command, THEN validation should pass.`` () =
+    let command =
+        {
+            EmailAddress = EmailAddress "anEmailAddress@fake.com"
+            Date = DateTime.MaxValue
+            OfficeId = office.Id
+        } : BookADesk
+    
+    let result = BookADeskReservationValidator.validateCommand offices command aReservationAggregate
+    match result with
+    | Error _ -> failwith "Validation should have succeeded"
+    | Ok _ -> ()
+    
+[<Fact>]
+let ``GIVEN A Book-A-Desk Reservation command with the user already booked, WHEN validating the command, THEN validation should fail.`` () =
+    let command =
+        {
+            EmailAddress = EmailAddress "anEmailAddress@fake.com"
```

Comment:
Extract the email to its own variable and reuse it in the reservation aggregate and command